installer/functions/plugin/helper-plugins.private.ts

```diff
@@ -0,0 +1,163 @@
+
+import axios from 'axios';
+import { ApplicationContext, Configuration } from '../../src/constants/interface';
+import { 
+  ACCOUNT_SID,
+  AUTH_TOKEN,
+  getParam
+} 
+from '../common';
+
+export const listPluginsEndpoint = "https://flex-api.twilio.com/v1/PluginService/Plugins";
+
+/**
+ * Needed to make API calls
+ * @param stringToConvert 
+ * @returns - base64 string
+ */
+export const convertToBase64 = (stringToConvert: string) => {
+  return Buffer.from(stringToConvert).toString('base64');
+}
+
+/**
+ * 
+ * @param accountSid 
+ * @param authToken 
+ * @returns - base64 User Credentials
+ */
+export const getBase64Credentials = (accountSid: string, authToken: string): string => {
+  return convertToBase64(accountSid + ":" + authToken);
+}
+
+/**
+ * Resource SID formats
+ * 
+ * Configurations = FJxxxxxxxxxxx;
+ * Versions = FVxxxxxxxxxx;
+ * Plugins = FPxxxxxxxxxxx;
+ * Releases = FKxxxxxxxxx;
+ */
+
+
+export const getAllPlugins = async (context: ApplicationContext) => {
+  const accountSid = await getParam(context, ACCOUNT_SID);
+  const authToken = await getParam(context, AUTH_TOKEN);
```

Comment:
again, sequential account fetches that really only need to fetch the first account

Reply:
the best case is to fetch the user account once, and then pass the account as a parameter through all of these functions